### PR TITLE
KG - Consolidated Request Tab Not Rendering

### PR DESCRIPTION
**Pull request by kayla-glick**
Note: Adding specs for this too. If they're not done in time, I will make another PR with them.

### CODE REVIEW

File: app/controllers/service_calendars_controller.rb [hunk 1]
```diff
@@ -89,13 +89,13 @@ def table
...
-    @merged       = true
-    @consolidated = false
-
+    @tab              = params[:tab]
+    @review           = params[:review] == 'true'
+    @portal           = params[:portal] == 'true'
+    @admin            = @portal && @sub_service_request.present?
+    @merged           = true
+    @consolidated     = false
+    @statuses_hidden  = []
```
Comment (wtholt) [inline app/controllers/service_calendars_controller.rb:98]: Maybe you could use #map here with the statuses hidden params

**Review by jleonardw9 — APPROVED** [review]

File: app/controllers/service_calendars_controller.rb (hunk 1 above)
Comment (ghost) [reply app/controllers/service_calendars_controller.rb:98]: `@statuses_hidden` is set again on line 115

Comment (wtholt) [reply app/controllers/service_calendars_controller.rb:98]: Yeah but he could instantiate the array with the params in one line, correct?. Unless I'm not seeing something.

Comment (ghost) [reply app/controllers/service_calendars_controller.rb:98]: There's no need for #map, since he's not iterating over an array.

**Review by jwiel86 — APPROVED** [review]

**Pull request merged**